### PR TITLE
Improve beehive message

### LINKED ISSUE #162: Bee hive interaction doesnt work.
suspected cause: 
https://jd.papermc.io/paper/1.19/org/bukkit/event/player/PlayerInteractEvent.html

> This event will fire as cancelled if the vanilla behavior is to do nothing (e.g interacting with air). For the purpose of avoiding doubt, this means that the event will only be in the cancelled state if it is fired as a result of some prediction made by the server where no subsequent code will run, rather than when the subsequent interaction activity (e.g. placing a block in an illegal position ([BlockCanBuildEvent](https://jd.papermc.io/paper/1.19/org/bukkit/event/block/BlockCanBuildEvent.html)) will fail.

https://github.com/hackclub/HCCore/blob/643ba6000a7f1f63c05d62a379ea521122ed5d93/src/main/java/com/hackclub/hccore/listeners/BeehiveInteractionListener.java#L18-L19

suggested fix: add ignoreCanceled=True or whatever

### PR DESCRIPTION
**Pull request by polypixeldev**
This PR fixes the issue where the message would not send, and also adds formatting to the message sent to the player.
Fixes #162

### CODE REVIEW

File: src/main/java/com/hackclub/hccore/listeners/BeehiveInteractionListener.java [hunk 1]
```diff
@@ -26,20 +28,29 @@ public void onPlayerInteract(final PlayerInteractEvent event) {
     }
 
     Block clickedBlock = event.getClickedBlock();
-    if (!(clickedBlock instanceof org.bukkit.block.Beehive)) {
+    if (clickedBlock == null) {
       return;
     }
-    int beeCount = ((org.bukkit.block.Beehive) clickedBlock).getEntityCount();
+    if (clickedBlock.getType() != Material.BEEHIVE) {
```
Comment (the456gamer) [inline src/main/java/com/hackclub/hccore/listeners/BeehiveInteractionListener.java:34]: ignores bees nests

**Review by the456gamer — CHANGES_REQUESTED** [review]
i did it by checking if clickedblock.getstate is instance of the block.beehive
(and doing a null check)
 but locking it to a material means that bee nests dont work, even tho they implement beehive

File: src/main/java/com/hackclub/hccore/listeners/BeehiveInteractionListener.java (hunk 1 above)
Comment (polypixeldev) [reply src/main/java/com/hackclub/hccore/listeners/BeehiveInteractionListener.java:34]: ah, ok. i'll do that

**Review by the456gamer — APPROVED** [review]
LGTM

**Comment by the456gamer** [comment]
tho ive been looking at new ways to do msgs, can you send a screenshot of what it looks like in this pr?

**Pull request merged**